CONTRIBUTING.md

```diff
@@ -296,6 +296,113 @@ OS and is built by Android NDK. This often introduces compatibility issues as
 Android (specifically Termux) is not a standard platform. Do not expect there
 are exist package recipes available out-of-box.
 
+## Commit guidelines
+
+A commit message should look something like:
+```
+<commitType>(<repo>/<package>): (The change made/Short description of the change)
+
+[An optional but **highly recommended** commit message describing the changes made in the commit]
+
+[Fixes (termux/repo)#<issue number>]
+[Closes (termux/repo)#<pr number>]
+```
+
+Any line in the commit **should not exceed 80 characters**. In case it does, consider using different wordings or language style which better summarizes the changes done.
+
+- `<commitType>` describes the type of commit. Types of commits:
```

Comment:
What about bug fixes (as in https://github.com/termux/termux-packages/commit/38bd77272ade88216ca1b6c2dde9b5e44feda96e just now for example)? `enhance` doesn't quite seem to describe that case

Reply:
I suggest to use `fix(scope): msg`. Scope can be anything already defined in guideline.

Reply:
A 'fix(scope): message' rule has been added as suggested by @MrAdityaAlok.

Hence resolving